src/pages/streaming/streaming.html

```diff
@@ -0,0 +1,16 @@
+<ion-header>
+  
+  <ion-navbar>
+    <ion-title>TV</ion-title>
+  </ion-navbar>
+  
+</ion-header>
+<ion-content padding>
+  <div class="video-container">
+    <video #player controls width="640" height="360" style="margin: auto;">
```

Comment:
I'm worried about fixing size of video player, how could it adapt on different screen sizes ? if screen.width is 320 or 1200 for example how it is displayed ?